forest_json_to_csv.go

```diff
@@ -67,18 +70,194 @@ func ForestJsonToCsv(config ForestJsonToCsvConfig) error {
 	return nil
 }
 
-// TODO(hank)
 func writeForestCsv(forests []Forest, path string) error {
+	// open file
+	csvFile, err := os.Create(path)
```

Comment:
just a general call out that there are multiple places in the code base that deal with the file and csv apis, we could make a single utility that was just 
```
func writeCSV(header []string. rows string[][]) error
```
and then each transform is just responsible for produce the new rows, but doesn't care about the CSV details